### ISSUE TITLE
customPath set to false and link alternate

### ISSUE THREAD

**Issue by existe-deja** [nuxt-modules/i18n#1177]
### Version
nuxt-i18n: 
nuxt: 

#### [mode](https://nuxtjs.org/guides/configuration-glossary/configuration-mode): 
 - [x] universal
 - [ ] spa

### Nuxt-i18n configuration
```
i18n: {
    langDir: "lang/lazy/",
    lazy: true,
    defaultLocale: "en",
    locales: [
      {
        code: "en",
        iso: "en-EN",
        file: "en.js",
        domain: "www.something.com"
      },
      {
        code: "en-US",
        iso: "en-US",
        file: "en-US.js",
        domain: "www.en-us.something.com"
      }
    ],
    strategy: "prefix_except_default",
    differentDomains: true,
    parsePages: false,
    pages: {
      about: {
        en: "/about-page",
        "en-US": false
      }
    },
    seo: false,
    vueI18n: {
      fallbackLocale: {
        default: ["en"]
      }
    }
  }
```

### Reproduction Link
https://codesandbox.io/s/nuxt-i18n-template-forked-bk64e?file=/nuxt.config.js

### Steps to reproduce
With the en lang (default) set, click on about page link and see the meta link.

### What is Expected?
if a page is disabled, it shouldn't be injected in the meta tags.

### What is actually happening?
As the doc says https://i18n.nuxtjs.org/seo#automatic-hreflang-generation:

> Generates <link rel="alternate" hreflang="x"> tags for every language configured in nuxt.config.js. The language's ISO codes are used as hreflang values.

So it injects a tag even if the route is disabed.

**Comment by existe-deja** [comment]
that's sad :/

**Comment by rchl** [comment]
This depends on #984 being solved.

**Comment by BobbieGoede** [comment]
Closing this as v7 is not being actively worked on (critical hotfixes only), if this still applies to v8 and later please open a new issue!
